### PR TITLE
Update doctrine.rst

### PR DESCRIPTION
**Pull request by hectorprats**
Uploading code of directly access in controller of the custom dbal connections.

### CODE REVIEW

**Comment by hectorprats** [comment]
I don't know what is wrong with that

File: reference/configuration/doctrine.rst [hunk 1]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
 ``default_connection`` parameter.
 
 Each connection is also accessible via the ``doctrine.dbal.[name]_connection``
-service where ``[name]`` is the name of the connection.
+service where ``[name]`` is the name of the connection. Or directly in the controller with 
+getConnection method and the name of the connection.
+
+.. code-block:: php
```
Comment (gregurco) [inline reference/configuration/doctrine.rst]: You have an error `"Please do not use ".. code-block:: php", use "::" instead."`. See the example how to insert php code in other files, for example here: https://raw.githubusercontent.com/symfony/symfony-docs/master/reference/configuration/kernel.rst

File: reference/configuration/doctrine.rst [hunk 2]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
 ``default_connection`` parameter.
 
 Each connection is also accessible via the ``doctrine.dbal.[name]_connection``
-service where ``[name]`` is the name of the connection.
+service where ``[name]`` is the name of the connection. Or directly in the controller with
```
Comment (gregurco) [inline reference/configuration/doctrine.rst]: I'm not sure that it's ok to start new sentence with `Or`.

File: reference/configuration/doctrine.rst [hunk 3]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
 ``default_connection`` parameter.
 
 Each connection is also accessible via the ``doctrine.dbal.[name]_connection``
-service where ``[name]`` is the name of the connection.
+service where ``[name]`` is the name of the connection. Or directly in the controller with 
+getConnection method and the name of the connection.
```
Comment (OskarStark) [inline reference/configuration/doctrine.rst]:
```suggestion
getConnection method and the name of the connection::
```

File: reference/configuration/doctrine.rst [hunk 4]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
...
+
```
Comment (OskarStark) [inline reference/configuration/doctrine.rst]:
```suggestion

```

File: reference/configuration/doctrine.rst (hunk 1 above)
Comment (OskarStark) [inline reference/configuration/doctrine.rst]:
```suggestion
.. code-block:: php
```

File: reference/configuration/doctrine.rst [hunk 5]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
...
+.. code-block:: php
+
+    $conex = $this->getDoctrine()->getConnection('customer');
```
Comment (OskarStark) [inline reference/configuration/doctrine.rst]:
```suggestion
    $conection = $this->getDoctrine()->getConnection('customer');
```

File: reference/configuration/doctrine.rst [hunk 6]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
...
+    $res = $conex->fetchAll('select * from table');
```
Comment (OskarStark) [inline reference/configuration/doctrine.rst]:
```suggestion
    $result = $connection->fetchAll('select * from table');
```

File: reference/configuration/doctrine.rst (hunk 1 above)
Comment (hectorprats) [reply reference/configuration/doctrine.rst]: Thank, I cant see any diference.

**Comment by hectorprats** [comment]
thank you @OskarStark . You have been a great help

File: reference/configuration/doctrine.rst (hunk 1 above)
Comment (OskarStark) [reply reference/configuration/doctrine.rst]: Sorry, I updated it

File: reference/configuration/doctrine.rst [hunk 7]
```diff
@@ -155,7 +155,13 @@ which is the first one defined or the one configured via the
...
 
 Each connection is also accessible via the ``doctrine.dbal.[name]_connection``
-service where ``[name]`` is the name of the connection.
+service where ``[name]`` is the name of the connection. Or directly in the controller with 
+getConnection method and the name of the connection::
+
+.. code-block:: php
+
+    $connection = $this->getDoctrine()->getConnection('customer');
+    $result = $connection->fetchAll('select * from table');
```
Comment (OskarStark) [inline reference/configuration/doctrine.rst]: SELECT name FROM customers

would fit better IMO

File: reference/configuration/doctrine.rst (hunk 1 above)
Comment (OskarStark) [reply reference/configuration/doctrine.rst]: Please remove this

File: reference/configuration/doctrine.rst (hunk 7 above)
Comment (hectorprats) [reply reference/configuration/doctrine.rst]: it's just an example. I don't know the table's name or any field name.

Comment (OskarStark) [reply reference/configuration/doctrine.rst]: Yes but “Select * ... “ shouldn’t be promoted

Comment (hectorprats) [reply reference/configuration/doctrine.rst]: ok

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
@javiereguiluz please give us a final review here. Thanks

**Comment by wouterj** [comment]
Thanks for adding this tip to the docs @hectorprats! I've merged this and rewrote the sentence a bit in 93ef5f2 to make clear that you have to extend `AbstractController` for this to work (that's not a requirement anymore).

**Pull request closed without merging**